### ISSUE TITLE
[enhancement] search

### ISSUE THREAD

**Issue by kyle-ilantzis** [nicholasrout/lens-launcher#9]
sometimes I can't find the app i am lookign for, a search icon would be helpful.

It could be part of the grid of app icons, a search icon that looks like an app icon ... 

but it lets you search through you installed apps

**Comment by nicholasrout** [comment]
Good idea, thanks @kyle-ilantzis. Will consider this for a future release

**Comment by seanauriti** [comment]
Would be good if it was swipe down to search

**Comment by MatthewScholefield** [comment]
Yes, swipe down to search would IMO be a very valuable feature. Right now I use a pie control with search as the only option in an attempt to simulate this.

**Comment by nicholasrout** [comment]
As a first step, I would like to add a search filter to the apps tab in settings. If this was around, would the having search on the lens view still be desired? All opinions welcome

**Comment by MatthewScholefield** [comment]
Absolutely, for me. Although it would be convenient to add a button to
launch a Google search with the typed search phrase.
